freqgen/api.py

```diff
@@ -99,3 +108,11 @@ def predict(
         playlist=PlaylistLinks(**model.get_best_playlist(best_station)),
         image=image,
     )
+
+class Analytics(BaseModel):
+    questionnaire_completed: int
+
+@app.get("/analytics/all")
+def get_analytics() -> Analytics:
+    count = get_count_questionnaires()
+    return Analytics(questionnaire_completed=count)
```

Comment:
Dernier changement (que je vais appliquer moi meme pour merge ensuite) : 
```suggestion
    return Analytics(
        questionnaire_completed=get_count_questionnaires()
    )
```